deepchem/models/callbacks.py

```diff
@@ -2,6 +2,7 @@
 Callback functions that can be invoked while fitting a KerasModel.
 """
 import sys
+from deepchem.models.wandblogger import WandbLogger
```

Comment:
ValidationCallback shouldn't have any explicit references to WandbLogger.  The idea of having a generic Logger class is that all common code (like ValidationCallback) will reference it through the generic API without needing to know exactly what kind of Logger it is.

Reply:
I'm importing this since I make a check for the logger type later on in the code.
This actually is something you touch on in another comment. We can continue the discussion down below:
https://github.com/deepchem/deepchem/pull/2577#discussion_r676854073